src/walletSdk/Horizon/Stellar.ts

```diff
@@ -73,4 +81,60 @@ export class Stellar {
       throw e;
     }
   }
+
+  async submitWithFeeIncrease({
+    sourceAddress,
+    timeout,
+    baseFeeIncrease,
+    operations,
+    signingAddresses,
+    baseFee,
+    memo,
+    maxFee,
+  }: SubmitWithFeeIncreaseParams): Promise<Transaction> {
+    const builder = await this.transaction({
+      sourceAddress,
+      timebounds: timeout,
+      baseFee,
+      memo,
+    });
+
+    operations.forEach((op) => {
+      builder.addOperation(op);
+    });
+
+    const transaction = builder.build();
+    if (signingAddresses?.length) {
+      signingAddresses.forEach((signer) => {
+        transaction.sign(signer.keypair);
+      });
+    } else {
+      transaction.sign(sourceAddress.keypair);
```

Comment:
If function is null AND sourceAddress is SigningKeyPair, we can default to 
`transaction.sign(sourceAddress.keypair);`
otherwise -> throw exception requiring to specify signing function

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/31/commits/aa70c042ae7b62225fac9122e83a9000c159aac5